src/components/Tab/Tab.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export default function Tab({ match, tabs }) {
```

Comment:
```suggestion
export default function Tab({ match: { params: { currentTabId } }, tabs }) {
```
можна зразу тут витягнути потрібні дані